PC02_Graffiti_FLETCHER.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: YOUR NAME
+May 29, 2020
+'''
+
+from turtle import * #import the library of commands that you'd like to use
+
+colormode(255)
+
+# Create a panel to draw on. 
+panel = Screen()
+w = 750 # width of panel
+h = 750 # height of panel
+panel.setup(width=w, height=h) #600 x 600 is a decent size to work on. 
+#You can experiment by making it the size of your screen or super tiny!
+
+# Create a colorful background and add Bezos image to it
+image = "Bezos.gif"
+panel.bgcolor("lightsteelblue1")
+panel.bgpic(image)
+
+#=======Add your code here======
+home() #turtle starts in middle of screen
+
+# left eye (his right)
+up()
+goto(-15,110)
+down()
+color('red')
+dot(10)
+
+# right eye (his left)
+up()
+left(13)
+forward(60)
+dot(10)
```

Comment:
Use the RightEye Turtle to call commands with dot notation
```suggestion
RightEye.up()
RightEye.goto(-15,110)
RightEye.left(13)
RightEye.forward(60)
RightEye.down()
RightEye.dot(10)
```